components/local_date_time/local_date_time.test.tsx

```diff
@@ -68,29 +68,36 @@ describe('components/LocalDateTime', () => {
     });
 
     test('should render date with timezone enabled', () => {
-        const wrapper = mountWithIntl(
-            <LocalDateTime
-                eventTime={new Date('Fri Jan 12 2018 20:15:13 GMT+0000 (+00)').getTime()}
-                enableTimezone={true}
-                timeZone={'Australia/Sydney'}
-            />
-        );
+        const baseProps = {
+            eventTime: new Date('Fri Jan 12 2018 20:15:13 GMT+0000 (+00)').getTime(),
+            enableTimezone: true,
+            timeZone: 'Australia/Sydney',
+        };
 
+        let wrapper = mountWithIntl(<LocalDateTime {...baseProps}/>);
         expect(wrapper.find('time').prop('title')).toBe('Sat Jan 13 2018 07:15:13 GMT+1100 (Australia/Sydney)');
-        expect(wrapper.find('span').text()).toBe('7:15 AM');
+        expect(wrapper.find('span').text()).toBe('07:15 AM');
+
+        wrapper = mountWithIntl(<LocalDateTime {...baseProps} timeZone={'US/Hawaii'}/>);
```

Comment:
Use of setProps is neat but it's failing on check-types when wrapped by `mountWithIntl`.